Register_Login/Register.js

```diff
@@ -75,9 +74,9 @@ function register({ navigation }) {
           <Image
             source={require('../assets/moprim.png')}
             style={{
-              width: 170,
-              height: 170,
-              borderRadius: 100,
+              width: 150,
```

Comment:
A good practice here should be like this.
```
const LOGO_SIZE = 150;
...
width: LOGO_SIZE,
height: LOGO_SIZE,
borderRadius: LOGO_SIZE / 2,
...
```

An even better and responsive approach is using sizes in `components/Spacing` for sizing. You may check the example in LeaderboardScreen

Reply:
ok thanks